### PR TITLE
chore(ci): Enable ips rpcnodes

### PR DESCRIPTION
**Pull request by sfrique**
## Purpose

Make circleci able to set rpc nodes load balancer

## For reviewers

This temporary is pointing to a different branch

or is for additional features. Patch is for bug fixes.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -321,9 +333,15 @@ jobs:
...
-            <tr><td>Boot Node 1</td><td>${PRNET_IPS[0]}</td></tr>\
-            <tr><td>Boot Node 2</td><td>${PRNET_IPS[1]}</td></tr>\
-            <tr><td>Boot Node 3</td><td>${PRNET_IPS[2]}</td></tr>\
+            <tr><td>Boot Node 1</td><td>${PRNET_IPS_BOOT_NODES[0]}</td></tr>\
+            <tr><td>Boot Node 2</td><td>${PRNET_IPS_BOOT_NODES[1]}</td></tr>\
+            <tr><td>Boot Node 3</td><td>${PRNET_IPS_BOOT_NODES[2]}</td></tr>\
+           </table>
+           <h4><p>RPC Nodes</p></h4>\
+           <table>\
+            <tr><td>Boot Node 1</td><td>${PRNET_IPS_RPC_NODES[0]}</td></tr>\
```
Comment (rattrap) [inline .circleci/config.yml:342]: Should also change the labels for each node

File: .circleci/config.yml [hunk 2]
```diff
@@ -368,6 +386,12 @@ jobs:
             <tr><td>Boot Node 1</td><td>${PRNET_IPS[0]}</td></tr>\
             <tr><td>Boot Node 2</td><td>${PRNET_IPS[1]}</td></tr>\
             <tr><td>Boot Node 3</td><td>${PRNET_IPS[2]}</td></tr>\
+           </table>
+           <h4><p>RPC Nodes</p></h4>\
+           <table>\
+            <tr><td>Boot Node 1</td><td>${PRNET_IPS_RPC_NODES[0]}</td></tr>\
```
Comment (rattrap) [inline .circleci/config.yml:392]: Same here

**Review by rattrap — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (sfrique) [reply .circleci/config.yml]: fixed in: 5362ae76

File: .circleci/config.yml (hunk 2 above)
Comment (sfrique) [reply .circleci/config.yml]: fixed in: 5362ae76

**Review by rattrap — APPROVED** [review]

**Review by JakubFornadel — APPROVED** [review]

**Comment by JakubFornadel** [comment]
@sfrique are we going to use this or it is not needed anymore ? If no, can we closed it ?

**Pull request merged**